`cargo build --release` is failing

**Describe the bug**
Latest commit from `main` branch build is failing. Sharing short version of logs here for ref.:

```bash
error[E0425]: cannot find value `node_config` in this scope
   --> src/models/src/rauthy_config.rs:163:31
    |
163 |             is_ha_deployment: node_config.nodes.len() > 1,
    |                               ^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `vars` in this scope
   --> src/models/src/rauthy_config.rs:174:13
    |
37  |     pub vars: Vars,
    |         ---- a field by that name exists in `Self`
...
174 |             vars,
    |             ^^^^
    |
help: consider importing one of these functions
    |
1   + use crate::rauthy_config::env::vars;
    |
1   + use std::env::vars;
    |
1   + use dotenvy::vars;
    |

error[E0425]: cannot find value `node_config` in this scope
   --> src/models/src/rauthy_config.rs:177:18
    |
177 |         Ok((slf, node_config))
    |                  ^^^^^^^^^^^ not found in this scope

For more information about this error, try `rustc --explain E0425`.
error: could not compile `rauthy-models` (lib) due to 31 previous errors
warning: build failed, waiting for other jobs to finish...
```

**To Reproduce**
Run `cargo build --release`

--- a/src/models/src/rauthy_config.rs
+++ b/src/models/src/rauthy_config.rs
@@ -44,7 +44,6 @@ impl RauthyConfig {
         tx_events: flume::Sender<Event>,
         tx_events_router: flume::Sender<EventRouterMsg>,
     ) -> Result<(Self, hiqlite::NodeConfig), Box<dyn Error>> {
-        #[cfg(debug_assertions)]
         let (vars, node_config) = Vars::load(config_file).await;
         vars.validate();
         if let Err(err) = node_config.is_valid() {

